docs/modules/ROOT/pages/reference/arch-control-plane.adoc

```diff
@@ -0,0 +1,162 @@
+
+The service catalog is manually defined on the CCP by adding the necessary configurations via a Project Syn Commodore Component.
+The XRD for each service on the CCP is exactly the same as the one on the CSPCP, the differentiation lies in the https://docs.crossplane.io/v1.18/concepts/compositions/[Composition^].
+This allows for true portability, the API spec per service is the same, no matter if the service is provisioned on the CCP or on a CSPCP.
+
+The Composition on the CCP wraps the Claim in an `Object` type from https://github.com/crossplane-contrib/provider-kubernetes[provider-kubernetes^].
+To specify at which CSP and Zone a service gets provisioned, the XRD exposes two fields:
+
+* `spec.servicelocation.serviceprovider`
+* `spec.servicelocation.serviceproviderzone`
+
+The field `spec.servicelocation.serviceprovider` is used on the CCP to choose the `spec.providerConfigRef.name` for `provider-kubernetes`, so that the Claim is transported to the correct CSP.
+On the CSPCP the field `spec.servicelocation.serviceproviderzone` is used to choose the `spec.providerConfigRef.name` for the worker cluster where the service will be provisioned in.
+
+In the <<Origin Metadata>> we specify a list of XRDs (e.g. `spec.XRDRefs[]`) which define the available services per CSP.
+We also configure the ProviderConfig ref in the origin metadata configuration.
+
+Access to list the available XRDs on cluster scope is denied to users of the CCP.
+See <<Listing>> on how the service catalog is made available.
+
+=== Listing
+
+The available service catalog is subject to the organization, and specifically it's origin.
+Therefore, the catalog is made available in the organization's context as a namespaced object called `catalog`.
+
+The object `catalog` is provided by a custom API server, which provides a filtered view on the available <<Service Catalog>>.
+Specifically, it's a filtered list of XRDs.
+It looks up the available XRDs in the <<Origin Metadata>> in the `spec.XRDRefs[]` field.
+
+* TODO: Another way: Have a `catalog` XRD and a controller which reconciles these objects into the Org Namespace, this way we wouldn't need an API server
+
+== Service Provisioning
+
+The CSPCP is an independent Control Plane which also could be used without the CCP.
+Therefore, the CCP acts like "a user" of the CSPCP and therefore places Claims into a organization namespace.
+This Claim is then reconciled into the service instance.
+
+[mermaid]
+....
+sequenceDiagram
+    autonumber
+    actor EU as User
+    participant CCP as Central Control Plane
+    participant CSPCP as CSP Control Plane
+    participant WRK as Worker Cluster / Zone
+
+    EU->>CCP: Create Claim
+    CCP->>CCP: Validate Claim
+    CCP->>CSPCP: Create Claim
+    CSPCP->>WRK: Create Instance
+    WRK->>CSPCP: Created
+    CSPCP->>CCP: Created
+    CCP->>EU: Created
+....
+
+* TODO: How are credentials synchronized from the CSPCP to the CCP for the end-user to see?
```

Comment:
Either a push or a pull approach I guess. I feel like we'll need bi-directional connection between the CCP and the CSPCPs.